src/components/colorpalette/ColorPalette.jsx

```diff
@@ -0,0 +1,15 @@
+import "./colorpalette.css";
+export function ColorPalette() {
+  return (
+    <>
+      <div className="children-center grid-gap card-palette">
+        <div className="shade-ball card-shade-1"></div>
+        <div className="shade-ball card-shade-2"></div>
+        <div className="shade-ball card-shade-4"></div>
+        <div className="shade-ball card-shade-3"></div>
+
+        <div className="shade-ball card-shade-5"></div>
```

Comment:
Can have a list of colors which you can map over 
```
[c1,c2,c3,c4,c5],map((everyColor,index)=>{
return (
<div key={everyColor} className=`shade-ball card-shade-${index+1}`></div>
)
})
```

Reply:
made change to use in an optimized manner, thanks!